src/Neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -64,22 +69,29 @@ public sealed class PolicyContract : NativeContract
         /// </summary>
         public const uint MaxStoragePrice = 10000000;
 
+        /// <summary>
+        /// The minimum block generation time that the committee can set in milliseconds.
+        /// </summary>
+        public const uint MinBlockGenTime = 1000;
```

Comment:
At least 100. N3 is more than two networks, committee should be able to run faster networks if/when needed.

Reply:
But better not have any limit at all. We may want 50ms in some scenarios.

Reply:
it's better to have a max

Reply:
Let's use `0` as a minimum bound and `30_000` as the maximum bound. Ref. https://github.com/nspcc-dev/neo-go/pull/3835#discussion_r1991406720

Reply:
This `MinBlockGenTime = 1000;` will be bad for privatenet tests and benchamarks,

Set this to 100

Reply:
This check is already added to `SetMSPerBlock` method.

So now `MinBlockGenTime` constant should be removed since it's unused.